### PR TITLE
cluster/Vagrantfile: ensure net.ipv4.ip_forward is set to 1 and warn Debian/Ubuntu users to set their FORWARD policy to ACCEPT

### PR DESCRIPTION
**Pull request by dseevr**
Signed-off-by: Bill Robinson <dseevr@users.noreply.github.com>

### CODE REVIEW

File: cluster/Vagrantfile [hunk 1]
```diff
@@ -141,6 +142,24 @@ if node_os == RHEL
...
+# access is required.
+if ARGV[0] == "up" && File.exists?("/etc/debian_version")
+  puts
+  puts "NOTE: Please ensure that the FORWARD chain's policy is set to ACCEPT"
+  puts "      by running: sudo iptables -P FORWARD ACCEPT"
+  puts
+end
+
+# if sysctl exists, ensure that ipv4 forwarding is enabled
+if find_executable0("sysctl") && %x{sysctl net.ipv4.ip_forward}.strip.chars.last == "0"
```
Comment (dseevr) [inline cluster/Vagrantfile]: `find_executable("foo")` prints "checking for foo... no" in addition to returning true/false

`find_executable0("foo")` just returns true/false

It looks like a typo 😄

File: cluster/Vagrantfile [hunk 2]
```diff
@@ -142,6 +141,26 @@ if node_os == RHEL
...
+# access is required.
+if ARGV[0] == "up" && File.exists?("/etc/debian_version")
+  puts
+  puts "NOTE: Please ensure that the FORWARD chain's policy is set to ACCEPT"
+  puts "      by running: sudo iptables -P FORWARD ACCEPT"
+  puts
+end
+
+# ensure that ipv4 forwarding is enabled if we're on Linux
+forward_file = "/proc/sys/net/ipv4/ip_forward"
```
Comment (gaurav-dalvi) [inline cluster/Vagrantfile]: is this supposed to be for non debian linux versions ?

Comment (dseevr) [inline cluster/Vagrantfile]: This is for all versions of Linux

**Comment by gaurav-dalvi** [comment]
Dont we need same stuff in our ansible playbooks : 

eg:
https://github.com/contiv/ansible/blob/master/roles/contiv_network/tasks/main.yml#L5

**Pull request merged**